### PR TITLE
add support for 'code' column in datatables

### PR DESCRIPTION
**Pull request by sergeiQ**

### CODE REVIEW

File: quandl/get_table.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import os
 
 
-def get_table(code, **options):
+def get_table(datatable_code, **options):
```
Comment (mbasset) [inline quandl/get_table.py]: My python must be rusty. Why does changing the variable from `code` to `datatable_code` fix this issue?

Comment (sergeiQ) [inline quandl/get_table.py]: we are passing filter conditions in options and column name can be "code". example `quandl.get_table("AT/DATA", code="my_code")`

Comment (mbasset) [inline quandl/get_table.py]: Then my only suggestion would to be to rename it to `quandl_code` rather than `datatable_code`

Comment (sergeiQ) [inline quandl/get_table.py]: quandl_code is reserved as a column for databases migrated from time-series

**Pull request merged**